src/types/global/index.d.ts

```diff
@@ -0,0 +1,11 @@
+namespace jest {
+  interface Matchers<R> {
+    toMatchScreenshots(): R
+  }
+}
+
+declare namespace NodeJS {
```

Comment:
Not sure we need this... I think you can set it at the tsconfig level: `"types": ["node"]`